package.json

```diff
@@ -6,9 +6,6 @@
   "repository": "https://github.com/artsy/reaction.git",
   "author": "Art.sy Inc <it@artsymail.com>",
   "license": "MIT",
-  "engines": {
-    "node": "^10.13.0"
-  },
```

Comment:
It'd probably be best if we didn't group this change in this PR. Also, I wonder if we shouldn't just introduce a range while we're migrating.  

```
"engines": {
	"node": ">=10.13.0 <13.0.0"
}
```

Reply:
I don't think it's necessary to have a lock here, we're not doing anything special other than running storybooks and compiling files with babel. If for some reason it causes problems we can add it back, but what i think this will do is remove problems and friction as client apps consume reaction.

Reply:
(Will move this to differnt PR tho!)

Reply:
Sounds good to me.